common/model-views.cpp

```diff
@@ -1157,6 +1158,7 @@ namespace rs2
         supported_options = s->get_supported_options();
         restore_processing_block("colorizer", depth_colorizer);
         restore_processing_block("yuy2rgb", yuy2rgb);
+        restore_processing_block("m4202rgb", m420_to_rgb);
```

Comment:
Can we change the function names from <format>2rgb into <format>_to_rgb?
I think it will be more readable as the format is ending with numbers

Reply:
Done